packages/nc-gui/composables/useMultiSelect/index.ts

```diff
@@ -106,6 +115,41 @@ export function useMultiSelect(
           if (typeof textToCopy === 'object') {
             textToCopy = JSON.stringify(textToCopy)
           }
+
+          if (columnObj.uidt === UITypes.Formula) {
+            textToCopy = textToCopy.replace(/\b(\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}\+\d{2}:\d{2})\b/g, (d: string) => {
```

Comment:
same as the previous comment.

And you can keep the regex in some modules and reuse it.